### ISSUE TITLE
Make EEGClassifier have very nice interface

### ISSUE THREAD

**Issue by robintibor** [braindecode/braindecode#48]
Put very good defaults so calling without anything will usually give you a good result.

**Comment by sliwy** [comment]
I just take a short look at current examples to see what can be improved in the EEGClassifier interface. I think that the most unfriendly part is the callbacks part which takes a lot of lines and does really similar thing for `CroppedDecoding` and `TrialDecoding` but differs in details. My idea is to create two list of standard callbacks in the `callbacks.py` file so we can just import `cropped_callbacks` or `trial_callbacks`. This way we will remove callback definition from examples. Do you think that we should hide more from the user? @robintibor I think that other things should be easily accessible by a user (like `batch_size`, `optimizer`, `lr`, `train_test_split`).

**Comment by robintibor** [comment]
I feel this may be a little too opaque for the user, how about we allow to supply a list of scikit-learn scoring functions like ['accuracy', 'f1_score'] either as `cropped_scorings` or as `trial_scorings` and rest of code is created internally? Then you can still decide which scorings you want but they are automatically correctly created and by default applied to both train and valid... ? How you think @sliwy

**Comment by sliwy** [comment]
Seems legit to me and won't need a lot of modification. I'll try to implement.

**Comment by sliwy** [comment]
@robintibor Do you know any way to infer if the callbacks should be `Cropped` or `Trial` in the `EEGClassifier`?

**Comment by sliwy** [comment]
@robintibor Take a look at it: https://github.com/sliwy/braindecode/commit/82f212698dc6ff003840c01ff1465d1e9a3faad7 it does not support `CroppedDecoding` for now. First version for `TrialDecoding`, should be better tested, now just to show the conception.

**Comment by robintibor** [comment]
Ya this seems really nice https://github.com/sliwy/braindecode/commit/82f212698dc6ff003840c01ff1465d1e9a3faad7. regarding knowing cropped or not, I think this can be a constructor_parameter `cropped=True/False`. I think it's not bad if this is written explicitly by user
